iocage_lib/ioc_start.py

```diff
@@ -213,17 +214,15 @@ def __start_jail__(self):
             }, _callback=self.callback,
                 silent=self.silent)
 
-        if self.conf['vnet'] and self.defaultrouter == 'none':
-            self.log.debug('Grabbing default route')
-            self.defaultrouter = self.get_default_gateway()[0]
-            self.log.debug(f'Default Gateway: {self.defaultrouter}')
+        if self.conf['vnet'] and self.defaultrouter == 'auto':
```

Comment:
While auto makes more sense, we need a migration in place for those who are already using none. DHCP and NAT users specifically will run into issues.

Reply:
Yeah, this is definitely a breaking change. I will provide a migration.